server/src/services/asset-v1.service.ts

```diff
@@ -245,36 +249,16 @@ export class AssetServiceV1 {
     return asset.previewPath;
   }
 
-  private async getLibraryId(auth: AuthDto, libraryId?: string) {
-    if (libraryId) {
-      return libraryId;
-    }
-
-    let library = await this.libraryRepository.getDefaultUploadLibrary(auth.user.id);
-    if (!library) {
-      library = await this.libraryRepository.create({
-        ownerId: auth.user.id,
-        name: 'Default Library',
-        assets: [],
-        type: LibraryType.UPLOAD,
-        importPaths: [],
-        exclusionPatterns: [],
-      });
-    }
-
-    return library.id;
-  }
-
   private async create(
     auth: AuthDto,
-    dto: CreateAssetDto & { libraryId: string },
+    dto: CreateAssetDto,
     file: UploadFile,
     livePhotoAssetId?: string,
     sidecarPath?: string,
   ): Promise<AssetEntity> {
     const asset = await this.assetRepository.create({
       ownerId: auth.user.id,
-      libraryId: dto.libraryId,
+      libraryId: null,
```

Comment:
This could be omitted here right? This should be possibly undefined at this point as it comes from AssetCreate which picks from AssetEntity

Reply:
Yeah it can be removed.